src/Word2vec/index.js

```diff
@@ -127,7 +127,7 @@ class Word2Vec {
     });
 
     // If none of the words in the set are found, also bail out
-    if (!miniModel.length) {
+    if (!Object.Keys(miniModel).length) {
```

Comment:
Requires update:
* Hi there! Thanks for this PR!
* I believe this approach makes sense, but there is a small typo. It can however be corrected as `Object.keys()` with the lowercase `keys` instead of the uppercase.